new/LIT6739PPDam.xml

```diff
@@ -0,0 +1,71 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6739PPDam" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer against hemorrhage (<foreign xml:lang="gez">dam</foreign>),
```

Comment:
What do you think about giving, in this particular case, `<foreign xml:lang="gez">ḥəmāma dam</foreign>` as equivalent to hemorrhage?

Reply:
yes, that's good!